frontend/components/common/BaseValueInput.vue

```diff
@@ -38,13 +41,17 @@ export default Vue.extend({
             default: undefined,
         },
         validator: {
-            type: Function as Vue.PropType<function>,
+            type: Function as Vue.PropType<Function>,
             default: () => {},
         },
         disabled: {
             type: Boolean,
             default: false,
         },
+        auctionType: {
```

Comment:
Can you please add new stories to reflect the changes and available options for this parameter 🤗. Thank you!

Reply:
I did not even see, we don't have a story for this component. Will have to add it during a repo cleanup sometime.